chromadb/api/types.py

```diff
@@ -122,11 +115,177 @@ def is_image(target: Any) -> bool:
     return True
 
 
-def maybe_cast_one_to_many_image(target: OneOrMany[Image]) -> Images:
-    if is_image(target):
-        return cast(Images, [target])
-    # Already a sequence
-    return cast(Images, target)
+class BaseRecordSet(TypedDict):
```

Comment:
Why does base record set not include metadata?

Reply:
This is meant to be stuff which is embeddable only. That way it's reusable for query.